CalcSolverSsr/src/components/BodyMassIndex.tsx

```diff
@@ -0,0 +1,95 @@
+import React, {useEffect, useState} from "react";
+
+const BodyMassIndex = () => {
+    useState()
+
+    const [weight, setWeight] = useState(0)
+    const [height, setHeight] = useState(0)
+    const [bmi, setBmi] = useState(0)
+    const [heightUnit, setHeightUnit] = useState("inches")
+    const [massUnit, setMassUnit] = useState("lbs")
+
+    useEffect(() => {
+        updateBmi()
+    })
+
+    const roundNumber = (num: number, dec: number) => {
+        return Math.round(num * Math.pow(10, dec)) / Math.pow(10, dec);
+    }
+
+    const updateBmi = () => {
+        let newWeight;
+        if (massUnit !== 'kg') {
```

Comment:
Try to use === instead of !== in general by inverting the if statement.

Reply:
Fixed. Is this just best practice or preference?

Reply:
Both. Negatives are an extra cognitive step. Instead of saying "equals" which is pretty straightforward, whoever is reading the code has to think "does not equal" which just demands extra attention.